sabnzbd/skintext.py

```diff
@@ -52,12 +52,12 @@
     "sch-rss_scan": TT("Read RSS feeds"),  #:  #: Config->Scheduler
     "sch-remove_failed": TT("Remove failed jobs"),  #: Config->Scheduler
     "sch-remove_completed": TT("Remove completed jobs"),  #: Config->Scheduler
-    "sch-pause_all_low": TT("Pause low prioirty jobs"),  #: Config->Scheduler
```

Comment:
These cannot be changed, it would remove all translations.
We correct English text through `en.po`, where these (and more) are already listed:
https://github.com/sabnzbd/sabnzbd/blob/develop/po/main/en.po